packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.base.tsx

```diff
@@ -119,6 +111,7 @@ export class SearchBoxBase extends React.Component<ISearchBoxProps, ISearchBoxSt
           placeholder={placeholderValue}
           onChange={this._onInputChange}
           onInput={this._onInputChange}
+          onBlur={this._onBlur}
```

Comment:
Will the input blur when the button gets focus?

Reply:
This behavior isn't changing. when button gets focus, onBlur and onFocus get called again. The blur is just moved from an event listener to an input callback

https://github.com/OfficeDev/office-ui-fabric-react/pull/11604/files#diff-c1170007a89461079892a4ffc0886f74L190

Reply:
Oh, got it. I'm not sure that's desirable, but if it's the same as the previous behavior, it's fine for now.

Reply:
yeah, i toyed around with a number of things to see if I could remove the double onFocus, but didn't have any luck. As you said, it's not a regression.